scrapy/core/downloader/contextfactory.py

```diff
@@ -28,9 +29,13 @@ class ScrapyClientContextFactory(BrowserLikePolicyForHTTPS):
          understand the SSLv3, TLSv1, TLSv1.1 and TLSv1.2 protocols.'
         """
 
-        def __init__(self, method=SSL.SSLv23_METHOD, *args, **kwargs):
+        def __init__(self, method=SSL.SSLv23_METHOD, ciphers_string=None, *args, **kwargs):
             super(ScrapyClientContextFactory, self).__init__(*args, **kwargs)
             self._ssl_method = method
+            if ciphers_string:
+                self._ssl_ciphers = AcceptableCiphers.fromOpenSSLCipherString(ciphers_string)
+            else:
+                self._ssl_ciphers = DEFAULT_CIPHERS
```

Comment:
`DEFAULT_CIPHERS` is a singleton so I kept it as it maybe improves the performance.

Reply:
I'm not sure it helps, as download handler always pass ciphers_string, which is "DEFAULT" by default.